integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1050,6 +1057,51 @@ public static void verifyAfterDeletion(Domain domain) throws Exception {
     k8sTestUtils.verifyNoClusterRoleBindings(domain1LabelSelector);
   }
 
+  /**
+   * create oracle db pod in the k8s cluster
+   *
+   * @param dbPropsFile - db properties file
+   * @return - OracleDB instance
+   * @throws Exception - if any error occurs when creating Oracle DB pod
+   */
+  public static OracleDB createOracleDB(String dbPropsFile) throws Exception {
```

Comment:
It will be nice if you can move this method to a new class DBUtils.java under the same package as TestUtils as TestUtils is growing.